modules/button/react/lib/ToolbarDropdownButton.tsx

```diff
@@ -0,0 +1,87 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {colors, spacing, borderRadius} from '@workday/canvas-kit-react-core';
+import {focusRing, useTheme, Themeable, EmotionCanvasTheme} from '@workday/canvas-kit-react-common';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {ButtonColors} from './types';
+import {ButtonContainer} from './parts';
+import {IconButtonProps} from './IconButton';
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+
+export interface ToolbarDropdownButtonProps
+  extends Omit<IconButtonProps, 'size' | 'variant' | 'toggled' | 'onToggleChange'>,
+    Themeable {}
+
+const ToolbarDropdownButton = ({
+  theme = useTheme(),
+  buttonRef,
+  'aria-label': iconArialabel,
+  icon,
+  children,
+  ...elemProps
+}: ToolbarDropdownButtonProps) => {
+  const containerStyles = {
+    padding: spacing.zero,
+    minWidth: spacing.l,
+    width: 'auto',
+    height: spacing.l,
+    borderRadius: borderRadius.m,
+    '& .wd-icon': {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      width: 20,
+      height: 20,
+    },
+  };
+
+  const chevronStyles = {
```

Comment:
is there a reason why we have the styles inside the function? they are getting recreated every render